projects/dotcms-field-elements/src/components/dot-form/dot-form.tsx

```diff
@@ -1,10 +1,10 @@
 import { Component, Element, Listen, Prop, State, Watch } from '@stencil/core';
```

Comment:
Avatar in the form should not be required.

BUT the button should be disabled while the image is uploading.